crates/config/src/config.rs

```diff
@@ -389,8 +392,12 @@ impl Default for FeeConfig {
         // CKB default weight is 1000 / 1000
         sudt_fee_rate_weight.insert(1u32.into(), 1000u64);
         Self {
-            meta_cycles_limit: 20000,     // 20K cycles
-            sudt_cycles_limit: 20000,     // 20K cycles
+            meta_cycles_limit: 20000, // 20K cycles
+            sudt_cycles_limit: 20000, // 20K cycles
+
+            /// 1176198 cycles in Polyjuice test case
+            eth_addr_reg_cycles_limit: 1_200_000,
```

Comment:
This value should estimate by gas used, since we convert gas_limit 1:1 to cycles_limit

Reply:
use unified 20K as `cycles_limit` for simple Godwoken native contracts